UserAdminBundle/Form/Type/UserType.php

```diff
@@ -13,13 +14,21 @@
 class UserType extends AbstractType
 {
     protected $class;
+    protected $languages;
 
     /**
      * @param string $class
      */
-    public function __construct($class)
+    public function __construct($class, Twig_Environment $twig)
```

Comment:
And if this parameter is empty, twig.globals.available_laguages will have fr and en as default value and none will be selectable in user form?